controlplane/cmd/nsmd/nsmd.go

```diff
@@ -14,11 +17,23 @@ import (
 func main() {
 	model := model.NewModel()
 
+	registryAddress := os.Getenv("K8S_REGISTRY_ADDRESS")
+	registryAddress = strings.TrimSpace(registryAddress)
+	if registryAddress == "" {
+		registryAddress = "localhost:5000"
+	}
+
+	registryConn, err := grpc.Dial("localhost:5000")
```

Comment:
Shouldn’t this be registryAddress rather than a literal?

Reply:
good catch